sympy/physics/control/lti.py

```diff
@@ -0,0 +1,1033 @@
+from sympy import (
+    Symbol, simplify, oo, exp,
+    Poly, lcm, LC, degree, Integral, integrate,
+    Matrix, eye, zeros, Basic, latex,
+    ShapeError, ImmutableMatrix, MutableMatrix,
+    SparseMatrix, MutableDenseMatrix
+)
+
+from sympy.printing import sstr
+
+# import mpmath for numercial results
+from mpmath import expm, quad, matrix as mpm_matrix
+
+__all__ = ['StateSpaceModel', 'TransferFunctionModel']
+
+_matrixTypes = (
+    Matrix, ImmutableMatrix, MutableMatrix, SparseMatrix, MutableDenseMatrix)
+
+
+class StateSpaceModel(Basic):
+    """State space model (ssm) of a linear, time invariant control system
+
+    Represents the standard state-space model with state matrix A, input matrix B, output matrix C, and
+    transmission matrix D. This makes the linear control system:
+        (1) x'(t) = A * x(t) + B * u(t);    x in R^n , u in R^k
+        (2) y(t)  = C * x(t) + D * u(t);    y in R^m
+    where u(t)  is any input signal, y(t) the corresponding output, and x(t) the system's state.
+
+    Parameters
+    ==========
+
+    *arg : TransferFunctionModel, Sympy-Matrices
```

Comment:
I think it would be better to make this explicit, instead of `*arg`, i.e. `StateSpaceModel(A, B, C=None, D=None)`. If you want to support passing in transfer functions too there are some other options:

```
tf = TransferFunction(...)
StateSpaceModel(*transferfunction_to_state_space(tf))
```

or

```
ssm = StateSpaceModel.from_transfer_function(tf)
```